scipy/stats/_continuous_distns.py

```diff
@@ -8854,6 +8854,206 @@ def _updated_ctor_param(self):
+
+    %(set_vals_stmt)s
+    >>> mean, var, skew, kurt = %(name)s.stats(%(shapes)s, moments='mvsk')
+
+    Display the probability density function (``pdf``):
+
+    >>> x = np.linspace(%(name)s.ppf(0.01, %(shapes)s),
+    ...                 %(name)s.ppf(0.99, %(shapes)s), 100)
+    >>> ax.plot(x, %(name)s.pdf(x, %(shapes)s),
+    ...        'r-', lw=5, alpha=0.6, label='%(name)s pdf')
+
+    Alternatively, the distribution object can be called (as a function)
+    to fix the shape, location and scale parameters. This returns a "frozen"
+    RV object holding the given parameters fixed.
+
+    Freeze the distribution and display the frozen ``pdf``:
+
+    >>> rv = %(name)s(%(shapes)s)
+    >>> ax.plot(x, rv.pdf(x), 'k-', lw=2, label='frozen pdf')
+
+    Check accuracy of ``cdf`` and ``ppf``:
+
+    >>> vals = %(name)s.ppf([0.001, 0.5, 0.999], %(shapes)s)
+    >>> np.allclose([0.001, 0.5, 0.999], %(name)s.cdf(vals, %(shapes)s))
+    True
+
+    Rather than using (``%(name)s.rvs``) to generate random variates, which is
+    very slow for this distribution, we can approximate the inverse CDF using
+    an interpolator, and then perform inverse transform sampling with this
+    approximate inverse CDF.
+
+    Using the shape parameters, obtain the bounds data that will be input to
+    ``cdf`` for later use in interpolation.
+
+    >>> a, b = %(name)s.support(k, v)
+    >>> a, b
+    (0, inf)
+
+    Note that the upper limit for the range for the ``cdf`` input is
+    technically infinite. Inputting 1 to the ``ppf`` gives infinity:
+
+    >>> %(name)s.ppf(1, k, v)
+    inf
+
+    To obtain a finite limit we can input a value that nears 1.
+
+    >>> b = %(name)s.ppf(.999, k, v)
+    >>> b
+    7.41058083802274
+
+    >>> from scipy.interpolate import interp1d
+    >>> xs = np.linspace(a, b, 50)
+    >>> cdf = %(name)s.cdf(xs, k, v)
+    >>> ppf = interp1d(cdf, xs, kind='next', fill_value='extrapolate')
+    >>> random_state = np.random.default_rng(123)
+    >>> r = ppf(random_state.uniform(size=1000))
+
+    And compare the histogram:
+
+    >>> ax.hist(r, density=True, histtype='stepfilled', alpha=0.2)
+    >>> ax.legend(loc='best', frameon=False)
+    >>> plt.show()
+
+    """
+
+    def _argcheck(self, k, v):
+        """Verify args"""
+        return np.all(k > 1) and np.all(v > 0)
+
+    def _munp(self, K, k, v):
+        _a, _b = self._get_support()
+        K = np.atleast_1d(K)
+
+        def _single_moment(K, k, v):
+            user_data = np.array([K, k, v], float).ctypes.data_as(
+                ctypes.c_void_p)
+
+            llc = LowLevelCallable.from_cython(
+                _stats,
+                '_genstudentized_range_moment',
+                user_data
+            )
+
+            return integrate.nquad(
+                llc,
+                ranges=[(-np.inf, np.inf), (0, np.inf), (_a, _b)],
+                opts=dict(epsabs=1e-11, epsrel=1e-12)
+            )[0]
+
+        return _single_moment(K, k, v)
```

Comment:
Is the formatting of this stuff copied from somewhere?
It looks like it probably satisfies PEP8 (please run `flake8` on everything and report back, BTW) but I wonder if it could be a little prettier.
For instance, for instance, could
```
 user_data = np.array([K, k, v], float).ctypes.data_as(
                ctypes.c_void_p)
```
be split into logical lines?

Reply:
That line is taken from `geninvgauss_gen._cdf`. They managed to fit it all on one line. 
https://github.com/mdhaber/scipy/blob/9ed05e7952493302f76652768d1d08a503544107/scipy/stats/_continuous_distns.py#L3568-L3579

It's possible to split it but that would necessitate (I think) introducing a single-use variable, which I generally try to avoid. 
Ex.
```
data_arr = np.array([K, k, v], float)
user_data = data_arr.ctypes.data_as(ctypes.c_void_p)
```
After checking it, making it a one-liner only goes past the limit by 3 chars, so shortening the name is a possibility, as is just making it a slightly too long one-liner. Opinions?

Reply:
I've tried to fix this by splitting (for example) `'_genstudentized_range_moment'` into its own variable `cython_symbol`, that both gives some clarification on what the string is and gets many of the split function calls onto one line.